### PR TITLE
PID: rename default PID tables and tasks

### PR DESCRIPTION
**Pull request by njacazio**
- Use the compressed nsigma tables as the default (no labels)
- The extended table is labelled as Full

- Rename extended PID response tasks with Full tag
- Rename tiny task without labels

### CODE REVIEW

**Comment by njacazio** [comment]
@jgrosseo what do you think?
`pidRespTOFEl` -> `pidTOFFullEl`
`pidRespTOFTEl` -> `pidTOFEl`

File: Analysis/Tasks/PID/pidTOF.cxx [hunk 1]
```diff
@@ -39,19 +39,19 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 
 #include "Framework/runDataProcessing.h"
 
-struct tofPid {
+struct tofPidTiny {
```
Comment (jgrosseo) [inline Analysis/Tasks/PID/pidTOF.cxx]: Why keep the tiny here?

File: Analysis/Tasks/PID/pidTOF.cxx [hunk 2]
```diff
@@ -159,7 +168,7 @@ struct tofPid {
   }
 };
 
-struct tofPidQa {
+struct tofPidTinyQa {
```
Comment (jgrosseo) [inline Analysis/Tasks/PID/pidTOF.cxx]: same

File: Analysis/Tasks/PID/pidTOFFull.cxx [hunk 3]
```diff
@@ -39,19 +39,19 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 
 #include "Framework/runDataProcessing.h"
 
-struct tofPidTiny {
+struct tofPid {
```
Comment (jgrosseo) [inline Analysis/Tasks/PID/pidTOFFull.cxx]: And this should be Full, right?

File: Analysis/Tasks/PWGLF/spectraTPCtiny.cxx [hunk 4]
```diff
@@ -62,9 +62,9 @@ struct tpcSpectraTiny {
   Filter collisionFilter = nabs(aod::collision::posZ) < cfgCutVertex;
   Filter trackFilter = (nabs(aod::track::eta) < cfgCutEta) && (aod::track::isGlobalTrack == (uint8_t) true);
   using TrackCandidates = soa::Filtered<soa::Join<aod::Tracks, aod::TracksExtra,
-                                                  aod::pidRespTPCTEl, aod::pidRespTPCTMu, aod::pidRespTPCTPi,
-                                                  aod::pidRespTPCTKa, aod::pidRespTPCTPr, aod::pidRespTPCTDe,
-                                                  aod::pidRespTPCTTr, aod::pidRespTPCTHe, aod::pidRespTPCTAl,
+                                                  aod::pidTPCEl, aod::pidTPCMu, aod::pidTPCPi,
+                                                  aod::pidTPCKa, aod::pidTPCPr, aod::pidTPCDe,
+                                                  aod::pidTPCTr, aod::pidTPCHe, aod::pidTPCAl,
```
Comment (jgrosseo) [inline Analysis/Tasks/PWGLF/spectraTPCtiny.cxx:67]: Why didn't the tiny task subscribe to the tiny tables before?

**Review by jgrosseo — COMMENTED** [review]
Looks good. See my minor comments inline.

File: Analysis/Tasks/PID/pidTOF.cxx (hunk 1 above)
Comment (njacazio) [reply Analysis/Tasks/PID/pidTOF.cxx]: You're right, I missed it ;) I will remove it

File: Analysis/Tasks/PWGLF/spectraTPCtiny.cxx (hunk 4 above)
Comment (njacazio) [reply Analysis/Tasks/PWGLF/spectraTPCtiny.cxx:67]: but it did, it was just badly written in the `T` suffix e.g. `pidRespTPCTTr`

**Comment by njacazio** [comment]
Ok, this is now fully implemented, I think we can go to tests!

**Review by jgrosseo — APPROVED** [review]

**Pull request merged**